projects/dsp-ui/src/lib/search/advanced-search/advanced-search.component.ts

```diff
@@ -124,6 +124,11 @@ export class AdvancedSearchComponent implements OnInit, OnDestroy {
     initializeOntologies(): void {
         this._dspApiConnection.v2.onto.getOntologiesMetadata().subscribe(
             (response: OntologiesMetadata) => {
+                response.ontologies = response.ontologies.filter(
+                    onto => onto.attachedToProject !== 'http://www.knora.org/ontology/knora-admin#SystemProject');
```

Comment:
instead of `http://www.knora.org/ontology/knora-admin#SystemProject` you can/should use the value from js-lib's Constants: `Constants.SystemProjectIRI`

Reply:
good point, fixed in cfc0583